python/lsst/summit/utils/blockUtils.py

```diff
@@ -236,6 +236,15 @@ def augmentData(self):
         but is also much harder to maintain/debug, as the vectorized regexes
         are hard to work with, and to know which row is causing problems.
         """
+        if 'lastCheckpoint' not in self.data.columns:
```

Comment:
Do you also need this check in `augmentDataSlow`?

Reply:
I actually tested and didn't, but it was because I was getting zero rows of data, and thus the `iterrows()` there was meaning that line never got hit, so you're right, if I only got few but non-zero lines that would need be necessary! Thanks - added 👍